roles/variables/defaults/main.yml

```diff
@@ -1400,9 +1400,9 @@ control_center_next_gen_dependency_prometheus_schema: "{{ 'https' if control_cen
 control_center_next_gen_dependency_prometheus_host: "{{ (groups.get('control_center_next_gen') if groups.get('control_center_next_gen') else ['localhost']) | confluent.platform.resolve_hostnames(hostvars) | first }}"
 control_center_next_gen_dependency_prometheus_port: 9090
 control_center_next_gen_dependency_prometheus_url: "{{control_center_next_gen_dependency_prometheus_schema}}://{{control_center_next_gen_dependency_prometheus_host}}:{{control_center_next_gen_dependency_prometheus_port}}"
-control_center_next_gen_dependency_prometheus_ssl_enabled: true
+control_center_next_gen_dependency_prometheus_ssl_enabled: false
 control_center_next_gen_dependency_prometheus_basic_auth_enabled: true
```

Comment:
shouldn't we also set basic auth default to false? the setup should default to no auth?